src/java.base/share/classes/java/time/chrono/ChronoLocalDateImpl.java

```diff
@@ -258,25 +258,6 @@ public D minus(long amountToSubtract, TemporalUnit unit) {
      */
     abstract D plusMonths(long monthsToAdd);
 
-    /**
-     * Returns a copy of this date with the specified number of weeks added.
-     * <p>
-     * This adds the specified period in weeks to the date.
-     * In some cases, adding weeks can cause the resulting date to become invalid.
-     * If this occurs, then other fields will be adjusted to ensure that the result is valid.
-     * <p>
-     * The default implementation uses {@link #plusDays(long)} using a 7 day week.
-     * <p>
-     * This instance is immutable and unaffected by this method call.
-     *
-     * @param weeksToAdd  the weeks to add, may be negative
-     * @return a date based on this one with the weeks added, not null
-     * @throws DateTimeException if the result exceeds the supported date range
-     */
-    D plusWeeks(long weeksToAdd) {
```

Comment:
Rather than removing it, this method should in fact be called from `plus(long amountToAdd, TemporalUnit unit)` to give subclasses the chance to override the 7 day behaviour. 

(Sure, this is an internal class, and there is no implementation that needs to do that, but it is how the class was designed, and it would be clearer to fix tha current mistaken setup)